README.md

```diff
@@ -151,19 +159,29 @@ You can run the website and the REST API that supports it locally. You need to f
                     "CompletionsMinTokens": 50,
                     "CompletionsMaxTokens": 300,
                     "SystemMaxTokens": 1500,
-                    "MemoryMinTokens": 500,
-                    "MemoryMaxTokens": 2500,
-                    "MessagesMinTokens": 1000,
-                    "MessagesMaxTokens": 3000
+                    "MemoryMinTokens": 1500,
+                    "MemoryMaxTokens": 700,
```

Comment:
Why is max tokens smaller than min tokens? @ciprianjichici

Reply:
Clearly a typo, should have been 7000. Fixed it.